### ISSUE TITLE
Any difference between sigmoid and softmax activation when using f_score and IOU metrics?

### ISSUE THREAD

**Issue by NIRVANALAN** [qubvel/segmentation_models.pytorch#44]
I am using UNet and res50 encoder now working on medical dataset with 7 classes. I noticed that my IOU score and f_score is pretty low during the whole training stage. Maybe it's because there exists class imbalance in my dataset and sometimes the background(which does not belongs to any classes) dominates an input image. Thanks for your work and I have 2 questions here, should I consider pixels with no labels as a class? Because in pixel_CE loss the target array should be 1 channel and all BG pixeles should be labeled with 0(a class number). Another question is what loss funciton should I use considering my dataset is unique and the IOU and f_score keeps low.. and how does the selection of activation functions influence the calculation of iou and f_score? 
Thanks

**Comment by NIRVANALAN** [comment]
also, I noticed in your segmentation_model(keras version) repo, there is [categorical_accuracy] metric and categorical_crossentropy loss. Have you implemented them in this repo via Pytorch? 
Thanks

**Comment by NIRVANALAN** [comment]
And another quick question, should I take all the unlabeld pixel(BG and others) as a separate class? And do torch.argmax(pred) during inference time? 

Thanks
